atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/impl/AbstractTableMappingService.java

```diff
@@ -95,30 +96,38 @@ private TableReference getFullTableName(TableReference shortTableName) {
 
     @Override
     public Set<TableReference> mapToFullTableNames(Set<TableReference> tableRefs) {
```

Comment:
Probably not, but wasn't sure what API guarantees we make with OSS atlas.  Went ahead and deleted the method since this is an API break already, but let me know if we should be doing something like \@Deprecated instead.